site/docs/operations/jdk-gc-choice.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Choice of JDK and GC
+description: Choosing the right JDK-GC combination is important.
+---
+
+The Java runtime has been evolving more rapidly in recent years and,
+after two decades, we finally got a new default garbage collector: the
+G1. Two more GCs are on their way to production and are available as
+experimental features: Oracle's Z and OpenJDK's Shenandoah. We at
+Hazelcast thought it was time to put all these new options to test and
+find which choices work well with distributed stream processing in
+Hazelcast Jet.
+
+Jet is being used for a broad spectrum of use cases, with different
+latency and throughput requirements. Here are three important
+categories:
+
+1. Low-latency unbounded stream processing, with moderate state. Example:
+  detecting trends in 500 Hz sensor data from 100,000 devices, and
+  sending corrective feedback within 10-20 milliseconds.
+2. High-throughput, large-state unbounded stream processing. Example:
+  tracking GPS locations of millions of users, inferring their velocity
+  vectors.
+3. Old-school batch processing of big data volumes. The relevant measure
+  is time to complete, which implies a high throughput demand. Example:
+  analyzing a day's worth of stock trading data to update the risk
+  exposure of a given portfolio.
+
+At the outset, we can observe the following:
+
+- in scenario 1 the latency requirements enter the danger zone of GC
+  pauses: 100 milliseconds, something traditionally considered as an
+  excellent result for a worst-case GC pause, may be a showstopper for
+  many use cases
+- scenarios 2 and 3 are similar in terms of demands on the garbage
+  collector. Less strict latency, but large pressure on the tenured
+  generation
+- scenario 2 is tougher because latency, even if less so than in
+  scenario 1, is still relevant
+
+We tried the following combinations:
+
+1. JDK 8 with the default Parallel collector and the optional G1
+2. JDK 11 with the default G1 collector
+3. JDK 14 with the default G1 as well as the experimental Z and
+  Shenandoah
+
+And here are our overall conclusions:
```

Comment:
Very nice, clean and concise, but maybe would be good to have a "where to go next" for some details for people who are interested. Anybody who finds any of your conclusions odd or doesn't agree with something has no way to "dig deeper" to see that you are actually right. Ie all the work and thoroughness you've put into these clean conclusions doesn't show. I know the conclusions are solid, because I've seen what's behind them, but simple readers won't.